airbyte-integrations/connectors/source-snowflake/unit_tests/integration/test_table.py

```diff
@@ -184,10 +185,10 @@ def setUp(self):
             "parent_stream": f"{_SCHEMA}.{_TABLE}",
             "where_clause": self.where_clause
         }
-        self.record = a_snowflake_response("response_get_table")
+        self.get_record_builder_instance = lambda: a_snowflake_response("response_get_table")
```

Comment:
Nice catch on seeing that we were modifying the same instance of the builder without creating a new one

nit: renaming this `self._a_record` would reduce noise when reading tests

Reply:
Done